src/vsr/superblock.zig

```diff
@@ -258,17 +276,17 @@ pub const SuperBlockSector = extern struct {
         if (!meta.eql(a.snapshots, b.snapshots)) return false;
         if (a.manifest_size != b.manifest_size) return false;
         if (a.free_set_size != b.free_set_size) return false;
-
-        assert(@bitCast(u32, a.reserved[0..4].*) == 0);
-        assert(@bitCast(u32, b.reserved[0..4].*) == 0);
```

Comment:
These aren't needed anymore since `reserved`'s size is now a multiple of 8-bytes.